### DISCUSSION TITLE
Python Wanted!

### DISCUSSION THREAD

**Discussion by TrroJay** [wiop93256/Simple-Code-Formmater#4] [Ideas]
What a fantastic plugin for Obsidian! We're excitedly looking forward to a new feature that formats Python code into prettier style.

**Comment by BXY1217** [comment]
Excuse me, what is the obsidian plug-in? I am a novice.

**Reply by TrroJay** [reply]
Obsidian is a note-taking software based on Markdown syntax. It is lightweight and suitable for daily note-taking.
